ui/main/src/app/modules/share/pipes/opfab-title-case.pipe.ts

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2022, Alliander (http://www.alliander.com)
+/* Copyright (c) 2018-2022, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+import {Pipe, PipeTransform} from '@angular/core';
+import {ConfigService} from '@ofServices/config.service';
+
+@Pipe({
+    name: 'opfabTitleCase'
+})
+export class OpfabTitleCasePipe implements PipeTransform {
+    constructor(private configService: ConfigService) {}
+
+    transform(title: string): string {
+        if (this.configService.getConfigValue('feed.card.titleUpperCase', true)) {
```

Comment:
This function is gonna be called several times so maybe, for performance reason, it would be better to get the parameter 'feed.card.titleUpperCase' in the constructor and put it in a global variable.